change/@fluentui-react-native-avatar-bde7ae22-cfb0-4643-8057-f7a10b48ccb1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think your fork of FURN is out of sync with upstream. When that happens, beachball makes a lot of unnecessary change messages. Could you double check that it's in sync and retry?